docs/INSTALLATION.md

```diff
@@ -1,6 +1,6 @@
 ## Installation
 
-Use the [installation guide at Jumpstart Lab](http://tutorials.jumpstartlab.com/topics/environment/environment.html) for instructions on installing Ruby.
+Choose your installation method at the [Ruby installation page](https://www.ruby-lang.org/en/documentation/installation/) for instructions on installing Ruby.
```

Comment:
Would it make sense to also recommend one of the methods? If I was installing Ruby for the first time I'd really like for someone to tell me about `rbenv` for example.